packages/postgres-database/src/simcore_postgres_database/models/api_keys.py

```diff
@@ -52,7 +53,7 @@
         nullable=False,
         doc="Identified product",
     ),
-    sa.Column("api_key", sa.String(), nullable=False),
+    sa.Column("api_key", sa.String(), nullable=False, index=True),
```

Comment:
I wonder whether what you need to index is rather (api_key,api_secret) pairs instead